cmd/api-errors.go

```diff
@@ -189,6 +189,7 @@ const (
 	ErrMalformedJSON
 	ErrAdminInvalidAccessKey
 	ErrAdminInvalidSecretKey
+	ErrAdminConfigNotificationTargetsFailed
```

Comment:
Always add new errors at the end - adding in between clobbers with git history.

Reply:
> Always add new errors at the end - adding in between clobbers with git history.

Just wanted to make sure to put Admin errors together, I didn't understand exactly what's the effect on git history.

Reply:
Primary problem is that adding errors in between is harder to track in `git blame` history. Adding at the end allows for lesser diff...